src/main/kotlin/sliv/tool/parsers/ParserUtils.kt

```diff
@@ -0,0 +1,17 @@
+package sliv.tool.parsers
+
+import java.io.File
+import java.io.FileNotFoundException
+
+object ParserUtils {
+    fun readFileText(filePath: String): String? {
+        var text: String? = null
+        try {
+            text = File(filePath).readText()
```

Comment:
Do we want to remove ability for sequential landmarks reading?

Reply:
At the moment, speed of the parsing is about 1.5 rows per second, so it isn't necessary, as I think. Even if it will be needed, we can add a second method "readFileSequentally", which returns Iterator and use it in the new Parser method "parseSequentally"